redbot/cogs/trivia/data/lists/sports.yaml

```diff
@@ -31,7 +31,7 @@ How many times has a Super Bowl went into overtime?:
 How many world records did swimmer Mark Spitz set when he won seven gold medals at the 1972 Olympics?:
 - 7
 - seven
-How much money is a US Olympic gold medalist awarded?:
+How much money is a US Olympic gold medallist awarded?:
```

Comment:
`medalist` is a correct US spelling:
https://dictionary.cambridge.org/us/dictionary/english/medalist